pallets/automation-time/src/lib.rs

```diff
@@ -71,11 +71,17 @@ pub mod pallet {
 	/// `MAX_TASK_WEGHT` + `RUN_TASK_LOOP_OVERHEAD`
 	const MAX_LOOP_WEIGHT: Weight = MAX_TASK_WEGHT + RUN_TASK_LOOP_OVERHEAD;
 
+	#[derive(Debug, Eq, PartialEq, Encode, Decode, TypeInfo)]
+	#[scale_info(skip_type_params(T))]
+	pub struct NotificationAction {
+		message: Vec<u8>,
+	}
+
 	/// The enum that stores all action specific data.
 	#[derive(Debug, Eq, PartialEq, Encode, Decode, TypeInfo)]
 	#[scale_info(skip_type_params(T))]
 	pub enum Action {
-		Notify(Vec<u8>),
+		Notify(NotificationAction),
```

Comment:
hmm I was just thinking of doing this in-line, but i guess a struct is fine

Reply:
curious how i even do this inline. looked into it a bit and it looks like structs cannot be anonymous

Reply:
same thing as the events right below it